snuba/datasets/configuration/events_analytics_platform/entities/eap_spans.yaml

```diff
@@ -23,7 +23,6 @@ schema:
     { name: name, type: String },
     { name: sampling_factor, type: Float, args: { size: 64 } },
     { name: sampling_weight, type: Float, args: { size: 64 } },
-    { name: sampling_weight_2, type: UInt, args: { size: 64 } },
```

Comment:
This PR seems to have weird merge conflict with https://github.com/getsentry/snuba/pull/6290/files#diff-438e988ddb8c60638398f11a2f2232da05c36cd13ed90683531bd73bc6418779R25-R27 - you can refer to the float version with sampling_weight above, no need to delete `sampling_weight_2` ?